lib/kbsecret/cli/kbsecret-env

```diff
@@ -37,6 +37,8 @@ selected_records = if cmd.opts.all?
                      end
                    end
 
+cmd.die "No such record." if selected_records.empty?
```

Comment:
Let's do `record(s)`, since the user could potentially query for more than one nonexistent record.